core/src/utilities/helpers/generic-helpers.js

```diff
@@ -257,11 +258,11 @@ class GenericHelpersClass {
   }
 
   getInnerHeight /* istanbul ignore next */() {
-    return LuigiElements.isCustomLuigiContainer() ? LuigiElements.getLuigiContainer().clientHeight : window.innerHeight;
+    return IframeHelpers.getIframeContainer().clientHeight;
   }
 
   getContentAreaHeight /* istanbul ignore next */() {
-    const contentAreaHeight = this.getInnerHeight() - this.getShellbarHeight();
+    const contentAreaHeight = this.getInnerHeight() - IframeHelpers.getIframeContainer().getBoundingClientRect().top;
```

Comment:
I think these changes, might need changing, since this works well when you have a main microfrontend iframe based. But when Having a WebComponent open and opening splitview, it gets distorted:
![Screenshot 2024-02-06 at 15 14 13](https://github.com/SAP/luigi/assets/18335489/98255335-12d0-420e-81a0-94a98d344858)